### PR TITLE
fead: add menu section

### PR DESCRIPTION
**Pull request by pellexa**
Добавил раздел Меню.
Деплой: https://lambda-restaurant-sp.netlify.app/

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -324,3 +324,106 @@ Colors:
...
+      rgb(255 255 255)
+    ),
+    url("../img/bg-menu.png");
+  box-shadow: inset 0 0 2.4rem 1.2rem rgb(0 0 0 / 6%);
+}
+
+.menu-content {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 14.6rem;
```
Comment (magicsd) [inline css/styles.css:345]: Why not just `gap`? `column-gap` belongs to multi-column layout.

File: css/styles.css [hunk 2]
```diff
@@ -324,3 +324,106 @@ Colors:
...
+}
+
+.menu-conten-column {
+  display: flex;
+  flex-direction: column;
```
Comment (magicsd) [inline css/styles.css]: If we'll just remove these properties, nothing happen

File: css/styles.css [hunk 3]
```diff
@@ -324,3 +324,106 @@ Colors:
...
+.menu-item-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu-dish,
+.menu-price {
+  font-family: "Yeseva One";
+  font-size: 2.2rem;
+  color: #cc9900;
```
Comment (magicsd) [inline css/styles.css]: `#cc9900` is used many times and could be implemented as CSS constant
```css
:root {
  --color-primary: #cc9900;
}
```
and then used as
```css
color: var(--color-primary);
```

File: css/styles.css (hunk 1 above)
Comment (pellexa) [reply css/styles.css:345]: This was before I made two columns, and then forgot to redo.

**Review by magicsd — APPROVED** [review]

**Pull request merged**